src/core/crypto.h

```diff
@@ -39,6 +39,13 @@ typedef struct QUIC_CRYPTO {
     //
     BOOLEAN CertValidationPending : 1;
 
+    //
+    // Indicates Resumption ticket validation is under validation asynchronously
+    //
+    BOOLEAN TicketValidationPending : 1;
+    BOOLEAN TicketValidationRejecting : 1;
+    uint32_t PendingValidationBufferLength;
```

Comment:
Let's put this in the Recv section of the struct, perhaps after `RecvTotalConsumed`.